frontend/src/scenes/onboarding/OnboardingProductConfiguration.tsx

```diff
@@ -29,22 +31,28 @@ type ConfigOption =
 interface PluginContent {
     title: string
     description: string
+    // some plugins make no sense e.g. geolocation doesn't apply to session replay
+    productOnboardingDenyList: ProductKey[]
 }
 type PluginContentMapping = Record<string, PluginContent>
 const pluginContentMapping: PluginContentMapping = {
     GeoIP: {
         title: 'Capture location information',
         description:
             'Enrich PostHog events and persons with IP location data. This is useful for understanding where your users are coming from. This setting can be found under the data pipelines apps.',
+        productOnboardingDenyList: [ProductKey.SESSION_REPLAY],
```

Comment:
This seems backwards to me, I feel like it should maybe be an allowlist?

Reply:
Do we not want the setting on all pages except those listed?

This felt clearer because we're saying "don't show only this one" over "do show on these other ones" and then you have to figure out how many it won't be shown on

No strong feeling here, since you all know this area better than me :)

Reply:
will merge it since it's green, @raquelmsmith happy to change to an allowlist if you have strong opinions.
but I agree with @pauldambra here, it's easier to add to a deny list rather than to an allowlist for all the new features  (looking at less maintenance)

Reply:
Sorry I forgot about this! 

I think for this specific instance the denylist makes sense. But as we move forward with other products that aren't so tied to product analytics (eg data warehouse) we'll have to remember to add those to the denylist. An allowlist is a better construct imo. But, we can change it in the future if / when it starts to get annoying. Thanks for getting this in!